src/Tribe/Editor/Attendee_Registration.php

```diff
@@ -124,7 +129,7 @@ public function render() {
 					$meta->accordion_content( $this->post->ID, $this->ticket_id );
 					?>
 				</div>
-				<button class="button-primary" type="submit">Save</button>
+				<button class="button-primary" type="submit"><?php echo __( 'Save Changes', 'event-tickets' ) ?></button>
```

Comment:
This should have escaping

```suggestion
				<button class="button-primary" type="submit"><?php esc_html_e( 'Save Changes', 'event-tickets' ) ?></button>
```